hera_mc/qm.py

```diff
@@ -0,0 +1,177 @@
+# -*- mode: python; coding: utf-8 -*-
+# Copyright 2017 the HERA Collaboration
+# Licensed under the 2-clause BSD license.
+
+"""Handling quality metrics of HERA data.
+
+"""
+from __future__ import absolute_import, division, print_function
+
+import numpy as np
+from astropy.time import Time
+from astropy.coordinates import EarthLocation
+from math import floor
+from sqlalchemy import (Column, Integer, BigInteger, Float, ForeignKey,
+                        String, ForeignKeyConstraint)
+from sqlalchemy.ext.hybrid import hybrid_property
+
+from hera_mc import geo_handling
+from . import MCDeclarativeBase, DEFAULT_GPS_TOL, DEFAULT_DAY_TOL, DEFAULT_HOUR_TOL
+
+
+class ant_metrics(MCDeclarativeBase):
```

Comment:
I'd much rather have these class name written in the standard `CamelCase` style. You can then put:

```
__tablename__ = 'ant_metrics'
```

To have the database table name be written in `snake_case`.